app/controllers/admin_controller.rb

```diff
@@ -7,32 +7,42 @@ def allow_only_admins
     # This shouldn't be needed becuse we also check for admin-ness in the
     # routes. Still doing this just to be safe.
     authenticate_user!
-    if not current_user.admin?
-      raise ActionController::RoutingError.new('Not Found')
-    end
+
+    fail ActionController::RoutingError 'Not Found' unless current_user.admin?
   end
 
   def find_or_create_by_mal
     media = params[:media].to_sym
     mal = MALImport.new(media, params[:mal_id].to_i, :shallow)
     @thing = case media
-    when :anime
-      Anime.create_or_update_from_hash mal.to_h
-    when :manga
-      Manga.create_or_update_from_hash mal.to_h
+             when :anime
+               Anime.create_or_update_from_hash mal.to_h
+             when :manga
+               Manga.create_or_update_from_hash mal.to_h
     end
     redirect_to @thing
     MyAnimeListScrapeWorker.perform_async(media, params[:mal_id].to_i)
   end
 
   def index
-    @anime_without_mal_id = Anime.where(mal_id: nil).where(%{"anime"."id" NOT IN (SELECT "anime_genres"."anime_id" FROM "anime_genres" INNER JOIN "genres" ON "genres"."id" = "anime_genres"."genre_id" AND "genres"."name" = 'Anime Influenced')})
+    @anime_without_mal_id = Anime.where(mal_id: nil).where(
+      %{"anime"."id" NOT IN (SELECT "anime_genres"."anime_id" \
+         FROM "anime_genres" INNER JOIN "genres" \
+         ON "genres"."id" = "anime_genres"."genre_id" \
+         AND "genres"."name" = 'Anime Influenced')}
+    )
     # Sort by partner code count (ascending) where there's less than 20 left
-    @deals = PartnerDeal.joins('left join partner_codes on partner_codes.partner_deal_id = partner_deals.id').
-      select('partner_deals.*, count(partner_codes.id) as codes_remaining').
-      group('partner_deals.id').
-      having('count(partner_codes.id) < 20').
-      order('count(partner_codes.id) asc')
+    @deals = PartnerDeal
+             .joins(
+               'LEFT JOIN partner_codes \
+                ON partner_codes.partner_deal_id = partner_deals.id'
+             )
+             .select(
+               'partner_deals.*, count(partner_codes.id) AS codes_remaining'
+             )
+             .group('partner_deals.id')
+             .having('count(partner_codes.id) < 20')
+             .order('count(partner_codes.id) asc')
     @blotter = Blotter.get
 
     if params[:old_kotodama].nil?
```

Comment:
What it's basically saying is "don't just wrap everything in an if statement when you mean to prevent it from running in a certain case"

In this case, though, the answer is probably removing the entire conditional — `old_kotodama` isn't used anymore (or ever, I think)